packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -52,6 +52,13 @@ export const generateTestDataForSlug = async (
     vite,
     templateFilepaths
   );
+
+  for (const [key, templateModule] of templateModuleCollection.entries()) {
+    if (templateModule.templateType !== "entity") {
+      templateModuleCollection.delete(key);
+    }
+  }
```

Comment:
If readability is a concern here, you could do something like [this](https://stackoverflow.com/a/53065133), although it's technically less efficient (shouldn't matter for the map sizes we're dealing with here though).

Reply:
Ack. The map way has just as many lines and really isn't easy to read, also at the cost of performance.